cypress/integration/WebInterface/Test Cases/MeasureHighlighting.spec.ts

```diff
@@ -0,0 +1,148 @@
+import {TestCaseJson} from "../../../Shared/TestCaseJson"
+import {CreateMeasurePage} from "../../../Shared/CreateMeasurePage"
+import {OktaLogin} from "../../../Shared/OktaLogin"
+import {MeasureGroupPage} from "../../../Shared/MeasureGroupPage"
+import {EditMeasurePage} from "../../../Shared/EditMeasurePage"
+import {TestCasesPage} from "../../../Shared/TestCasesPage"
+import {Utilities} from "../../../Shared/Utilities"
+import {MeasuresPage} from "../../../Shared/MeasuresPage"
+
+let measureName = 'TestMeasure' + Date.now()
+let CqlLibraryName = 'TestLibrary' + Date.now()
+let measureScoringArray = ['Ratio', 'Cohort', 'Continuous Variable', 'Proportion']
+let testCaseTitle = 'test case title'
+let testCaseDescription = 'DENOMFail' + Date.now()
+let testCaseSeries = 'SBTestSeries'
+let testCaseJson = TestCaseJson.TestCaseJson_Valid
+
+describe('Measure Highlighting', () => {
+
+
+    beforeEach('Create Measure and Login', () => {
+
+        let randValue = (Math.floor((Math.random() * 1000) + 1))
+        let newMeasureName = measureName + randValue
+        let newCqlLibraryName = CqlLibraryName + randValue
+
+        //Create New Measure
+        CreateMeasurePage.CreateQICoreMeasureAPI(newMeasureName, newCqlLibraryName, measureScoringArray[0])
+
+        OktaLogin.Login()
+
+    })
+    afterEach('Logout', () => {
+
+        OktaLogin.Logout()
+
+    })
+
+    afterEach('Clean up', () => {
+
+        let randValue = (Math.floor((Math.random() * 1000) + 1))
+        let newCqlLibraryName = CqlLibraryName + randValue
+
+        Utilities.deleteMeasure(measureName, newCqlLibraryName, measureScoringArray[0])
+
+    })
+
+    it('Execute single Test Case and verify Measure highlighting', () => {
+
+        //Add Measure Group
+        MeasureGroupPage.createMeasureGroupforRatioMeasure()
+
+        //Navigate to Test Cases page and add Test Case details
+        cy.get(EditMeasurePage.testCasesTab).click()
+        cy.get(TestCasesPage.newTestCaseButton).should('be.visible')
+        cy.get(TestCasesPage.newTestCaseButton).should('be.enabled')
+        cy.get(TestCasesPage.newTestCaseButton).click()
+
+        cy.get(TestCasesPage.testCasePopulationList).should('be.visible')
+
+        cy.wait(2000)
```

Comment:
do we need this wait?

Reply:
Copied from a different test which has wait. May be this is added to resolve the test failures

Reply:
lets see if it works without it